dune-project

```diff
@@ -66,7 +66,8 @@ possible and does not make any assumptions about IO.
   (csexp (>= 1.5))
   (ocamlformat-rpc-lib (>= 0.21.0))
   (odoc :with-doc)
-  (ocaml (and (>= 4.14) (< 4.15)))))
+  (ocaml (and (>= 4.14) (< 4.15)))
+  merlin-lib))
```

Comment:
There should be minimum version here I believe.

Reply:
Yes, also I removed the constraint on OCaml version, since it is now driven by the versions supported by merlin-lib